js/main.js

```diff
@@ -45,15 +45,20 @@
 
 
   var ui = {
+    onclick: function(selector, handler) {
+      [].forEach.call(document.querySelectorAll(selector), function(element) {
```

Comment:
On some browsers, the object returned by `querySelectorAll` is still not an iterable, so you have to trick it like that.